### PR TITLE
[Feature]: validation and toast message

### PR DESCRIPTION
**Pull request by 777Prabhjot**
Feat #79

##Description

Added validation for user id field and showed toast message with animation for error message.

### CODE REVIEW

File: client/src/views/Chat.vue [hunk 1]
```diff
@@ -334,6 +350,46 @@ export default {
...
+  color: #fff; /* White text color */
+  text-align: center; /* Centered text */
+  border-radius: 2px; /* Rounded borders */
+  padding: 16px; /* Padding */
+  position: fixed; /* Sit on top of the screen */
+  z-index: 1; /* Add a z-index if needed */
+  left: 50%; /* Center the snackbar */
+  top: 30px; /* 30px from the bottom */
+  -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
+  animation: fadein 0.5s, fadeout 0.5s 2.5s;
```
Comment (PradyumnaKrishna) [inline client/src/views/Chat.vue:369]: Remove the comments.

File: client/src/views/Chat.vue [hunk 2]
```diff
@@ -248,6 +263,7 @@ export default {
       const parsed = JSON.stringify(this.users);
       localStorage.setItem("users", parsed);
     },
+
```
Comment (PradyumnaKrishna) [inline client/src/views/Chat.vue:266]: Remove unexpected line addition.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
Looks good @777Prabhjot
Can you move this out to a separate component, in components directory?
Minor changes required.

**Comment by 777Prabhjot** [comment]
@PradyumnaKrishna done

**Review by PradyumnaKrishna — APPROVED** [review]
Thanks @777Prabhjot for your work.
LGTM

**Pull request merged**